app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,8 @@
+/*
+ *= require_tree .
+ *= require_self
+ *= link application.css
+ */
+
+@import 'bootstrap';
+@import 'bootstrap-icons/font/bootstrap-icons';
```

Comment:
- [ ] This error is raised When I try to precompile the assets using this command `rails assets:precompile`. Please fix that.

![image](https://github.com/ometman/hello-rails-react/assets/68796232/6860a863-f0d3-441d-b1ff-4ed505c42bca)

[Hint] This problem happened because you didn't configure `bootstrap` the correct way, You can remove it entirely or configure it the correct way.